### ISSUE TITLE
[x264:x64-windows] build failure

### ISSUE THREAD

**Issue by abcping123** [microsoft/vcpkg#13327]
**Host Environment**
- OS: Windows 10 v1903
- Compiler: Visual Studio 2017

**To Reproduce**
`./vcpkg install x264:x64-windows`

It could not cofigure at first, and I find the cause in `vcpkg_configure_make.cmake (line 223)`:
`string(REPLACE ";$ENV{SystemRoot}\\System32;" ";${MSYS_ROOT}/usr/share/automake-1.16;${MSYS_ROOT}/usr/bin;$ENV{SystemRoot}\\System32;" NEWPATH "$ENV{PATH}")`

This intends to add some paths, however `$ENV{PATH}` of my system only contains `C:\Windows\system32`, not `C:\Windows\System32`, so it fails.

After I modify this, it configures and builds well, but fails again when installing. 

[install-x64-windows-dbg-err.log](https://github.com/microsoft/vcpkg/files/5166962/install-x64-windows-dbg-err.log)
[install-x64-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/5166964/install-x64-windows-dbg-out.log)

It seems to be config errors, I have no idea how to fix it after trying some ways, so I come here to seek some help.

**Comment by abcping123** [comment]
By the way, environment variables in system setting contains exactly `C:\Windows\System32`, not `C:\Windows\system32`.

**Comment by sfhacker** [comment]
There seems to be a lot of issues building this port (x264). I wonder what's going on!
Just downloaded master branch and no issues at all.

![image](https://user-images.githubusercontent.com/25817529/92628383-107ee780-f310-11ea-8861-94894b5c675f.png)

@abcping123 Let us know if we can be of any assistance.

**Comment by NancyLi1013** [comment]
@Neumann-A 
Could you please help take a look? 
It seems that there are still some problems when building `x264 `after PR #11836 merged.

config-x86-wiindows-dbg-out.log
```
Unknown option --build=x86_64-pc-mingw32, ignored
Unknown option --disable-silent-rules, ignored
Unknown option --verbose, ignored
Unknown option --disable-static, ignored
Unknown system , edit the configure
```
config-x86-wiindows-dbg-err.log
```
./../src/14d026c3bd-bca2ebf740/configure: line 327: rm: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 328: rm: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 344: dirname: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 341: dirname: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 345: grep: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 554: /config.sub: No such file or directory
./../src/14d026c3bd-bca2ebf740/configure: line 565: basename: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 148: rm: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 167: cat: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 171: cat: command not found
./../src/14d026c3bd-bca2ebf740/configure: line 1: rm: command not found
```

**Comment by longnguyen2004** [comment]
I think it's time for us to switch to [ShiftMedia](https://github.com/ShiftMediaProject/x264)

**Comment by ZhengzhongSun** [comment]
I met the same questions now. Is there a stable commit of vcpkg which can be used now?

**Comment by Neumann-A** [comment]
@longnguyen2004 No, there is no need to do that.

@NancyLi1013 That seems to be an error introduced by @ras0219 changes to MSYS. (see https://github.com/microsoft/vcpkg/issues/13327#issue-691726380). It seems like x264 was  forgotten to be removed from the skiplist.

**Comment by ras0219-msft** [comment]
This should be fixed by #13450.

**Comment by longnguyen2004** [comment]
Alright I'll test it out
